stringvalidator/urn.go

```diff
@@ -24,7 +24,7 @@ Null (unconfigured) and unknown (known after apply) values are skipped.
 */
 func IsURN() validator.String {
 	return &common.RegexValidator{
-		Desc:         "must be a valid URN",
+		Desc:         "must be a valid URN. Now this validator is deprecated, please use `format.IsURN` instead",
```

Comment:
If you change the description, all schemas that use this validator will have their descriptions changed.